apps/meteor/tests/end-to-end/api/livechat/10-departments.ts

```diff
@@ -274,7 +274,8 @@ describe('LIVECHAT - Departments', function () {
 				.then(() => createDepartment())
 				.then((department: ILivechatDepartment) => {
 					dep1 = department;
-				}).then(() => createDepartment())
+				})
+				.then(() => createDepartment())
 				.then((department: ILivechatDepartment) => {
 					dep2 = department;
 				})
```

Comment:
Can't this happen in parallel?

Reply:
I think it's better to wait for the first department to already exist before creating another, this way we can make sure that the user can create more than one department